### ISSUE TITLE
Localize the dashboard layout on slow connections

### ISSUE THREAD

**Issue by pixdemo** [KeypDemoCorp/awesome-ui#625]
Accessibility should be considered when implementing this.

**Comment by pixdemo** [comment]
Any update on this? Happy to help if it's stuck.
